### PR TITLE
file handeling. validation. load config

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR changes the validation of the fluffy result file and cganges the cli to read a "load_config" data.

- Need input from @patrikgrenfeldt about load of concentratins and project name.
- Need handle lower case and upper case chr/Chr keys
- Neither Flowcell nor sampleProject should be requiered. Ie. Batch ID defined indepentently of fluffy csv file.  

**Review:**
- [x] code approved by @Mropat 
- [x] tests executed by @mayabrandi 
- [x] "Merge and deploy" approved by @mayabrandi 
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
@mayabrandi  and @J35P312 Should you (Maya) adjust to "wierd fluffy formats" as in commit (8d276fb) or would it be possible to make it more consistent in fluffy to allow easier parsing downstream?

**Comment by mayabrandi** [comment]
I already asked @J35P312 <https://github.com/J35P312> about that and the anser was that I have to adjust.

**Comment by henrikstranneheim** [comment]
Alright then! Good that you already discussed it

**Review by Mropat — APPROVED** [review]
:fire: 
Very beautiful work Maya!
I tried very hard to find something :stuck_out_tongue: 

However only can offer a small comment on one of the tests

File: tests/parse/test_parse_batch.py [hunk 1]
```diff
@@ -3,25 +3,25 @@
...
-    # WHEN running parse_batch_file	
+    # WHEN running parse_batch_file
+    results = parse_batch_file(invalid_csv)	
 
-    # THEN assert FileValidationError
-    with pytest.raises(FileValidationError):	
-        parse_batch_file(invalid_csv)
+    # THEN assert results is a list and it has length 3
+    assert isinstance(results, list)
+    assert len(results)==3
```
[the remark this answers is not in the document]
Comment (mayabrandi) [reply tests/parse/test_parse_batch.py]: So true. I was lazey. Pushed a test that will better check that the validation is doing its jobb

**Comment by mayabrandi** [comment]
Thanks for reviewing Maria!

**Pull request merged**